conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
```

Comment:
What is that?

Reply:
[Grayskull](https://github.com/conda/grayskull) is a conda recipe generator.  I [had been told](https://github.com/conda-forge/conda-forge.github.io/issues/2062#issuecomment-1890753604) that adding this line this was a way to help improve dependency updating for conda-forge recipes.  However, actually, looking at [this](https://conda-forge.org/docs/maintainer/conda_forge_yml/#bot), perhaps I should have set this to `update-all`